internal/pkg/flink/internal/controller/.snapshots/TestStatementControllerTestSuite-TestExecuteStatementPrintsUserInfo

```diff
@@ -1,5 +1,6 @@
 Statement name: test-statement
 Statement successfully submitted.
-Fetching results...
+Waiting for statement to be ready. Statement phase is COMPLETED.
+Statement phase is COMPLETED.
```

Comment:
Any reason this is printed twice?

Reply:
Oh, true. The test was just non-realistic, returning completed for a select query - selects always return pending. I've updated the test.